runtime/j9vm/javanextvmi.c

```diff
@@ -115,9 +115,10 @@ JVM_GetExtendedNPEMessage(JNIEnv *env, jthrowable throwableObj)
 			&& (NULL != userData.romMethod)
 			&& (UDATA_MAX != userData.bytecodeOffset)
 		) {
+			PORT_ACCESS_FROM_VMC(vmThread);
+			J9NPEMessageData *npeMsgData = (J9NPEMessageData *)j9mem_allocate_memory(sizeof(J9NPEMessageData), OMRMEM_CATEGORY_VM);
```

Comment:
Why malloc this instead of just using the struct as the local (i.e. no pointer)?

Reply:
Fixed.